### DISCUSSION TITLE
Has anyone had luck changing the slug for the posts folder?

### DISCUSSION THREAD

**Discussion by KeriGerheart** [colbyfayock/next-wordpress-starter#485] [General]
I tried copying all the files in the posts folder and changing this:
`export default function Posts({ posts, pagination }) {
  const title = `All Posts`;
  const slug = 'resources';`

and 
`  return {
    props: {
      posts,
      pagination: {
        ...pagination,
        basePath: '/resources',
      },
    },
  };
}`

in the [page].js file, but it doesn't seem to work. Has anyone done this successfully and could share what I missed here? Thanks!

**Comment by KeriGerheart** [comment]
Sorry, this was a dumb question. In case anyone else comes across it, the answer was to change postPathBySlug in lib/posts to include the desired folder name.

**Reply by colbyfayock** [reply]
glad you got it figured out!
